File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/properties/SetParticularProperty.java
package com.aspose.cells.cloud.examples.properties;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class SetParticularProperty {

    public static void main(String... args) throws IOException {
        // ExStart: set-particular-property
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "sample1.xlsx";
            Path inputFile = Utils.getPath(SetParticularProperty.class, input);
            String output = "sample2.xlsx";
            Path outputFile = Utils.getPath(SetParticularProperty.class, output);

            storageApi.PutCreate(input, Utils.STORAGE, null, inputFile.toFile());

            String propertyName = "Author";
            com.aspose.cells.model.CellsDocumentProperty body = new com.aspose.cells.model.CellsDocumentProperty();
            body.setName("AsposeAuthor");
            body.setValue("Aspose Plugin Developer");
            body.setBuiltIn("false");

            cellsApi.PutDocumentProperty(input, propertyName, Utils.STORAGE, Utils.FOLDER, body);

            com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload(input, null, Utils.STORAGE);

            Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: set-particular-property
    }

}

File: Examples/DotNet/CSharp/Document-Properties/SetParticularProperty.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Document_Properties
{
    class SetParticularProperty
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            // Set input file name
            String fileName = "Sample_Book1.xlsx";
            String propertyName = "AsposeAuthor";
            String storage = "";
            String folder = "";
            CellsDocumentProperty body = new CellsDocumentProperty();
            body.Name = "AsposeAuthor";
            body.Value = "Aspose Plugin Developer";
            body.BuiltIn = "false";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to set particular property
                CellsDocumentPropertyResponse apiResponse = cellsApi.PutDocumentProperty(fileName, propertyName, storage, folder, body);

                if (apiResponse != null)
                {
                    Console.WriteLine("New values for the Property have been changed!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
